### ISSUE TITLE
Arrays : render an empty array by default without any element inside it

### ISSUE THREAD

**Issue by enlight3d** [hamzahamidi/ajsf#256]
**Describe the bug**
which template: 

* [X] MaterialDesignFrameworkModule — Material Design
* [ ] Bootstrap3FrameworkModule — Bootstrap 3
* [ ] Bootstrap4FrameworkModule — Bootstrap 4
* [ ] NoFrameworkModule — plain HTML
* [ ] Other (please specify below)
  
I'm searching for a way to have an array inside an object but which would not be of type "required" and that would not be populated by default with one element until you click on the "add" button.

**To Reproduce**

N/A

**Expected behavior**

Something like "minItems" defined in the schema part that would work with also the fact that the key being present in "required" array or not to auto populate the array with one element or not.

**Desktop (please complete the following information):**

- OS: OSX
- Browser : Chrome
- Version : 88.0.4324.182
- Angular Version : 11
- AJSF Version : 0.4.0-beta.2

**Additional context**
By default we have this : 
<img width="1680" alt="Capture d’écran 2021-03-08 à 16 57 15" src="https://user-images.githubusercontent.com/19186686/110346158-91a10800-802f-11eb-90d7-940edd56825b.png">

I'm trying to have this :
<img width="1680" alt="Capture d’écran 2021-03-08 à 16 57 21" src="https://user-images.githubusercontent.com/19186686/110346247-a7163200-802f-11eb-88f9-6a3b8fbb93f7.png">

**Comment by SeekSwift** [comment]
Im also looking for this. @enlight3d did you ever find a way?

**Comment by enlight3d** [comment]
> Im also looking for this. @enlight3d did you ever find a way?

nope, I abandoned using this as it was too restrictive for my needs 😞
